__tests__/app.test.js

```diff
@@ -65,6 +65,60 @@ describe('GET /api/articles', () => {
         .expect(404)
 
     });
+    it('returns 200 status and queried articles but topic when endpoint contains a topic query', () => {
+        return request(app)
+        .get("/api/articles?topic=cats")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles[0]).toHaveProperty('topic', 'cats')
```

Comment:
Maybe add a .foreach to check that everything inside the articles array will have the property of topic with the value of cats? You'll have to check that the length is greater than 0 before the .foreach starts though!